app/clients/freshdesk.py

```diff
@@ -31,14 +31,25 @@ def _generate_description(self):
                 f'- main use case: {self.contact.main_use_case}',
                 f'- main use case details: {self.contact.main_use_case_details}',
             ])
+        elif self.contact.is_go_live_request():
+            message = '<br>'.join([
```

Comment:
I see that we use 2 `<br>` for demo requests, why is it different? What is the right thing?

Reply:
you are right, the question is why is the demo request different than the request to go live.
If you remember we moved the message generation from notification-admin into notification-api.
Functionally the message should be identical from the previous implementation going to freshdesk for the `demo` and `request to go live`. If we want to consider change the format of the message this is a change in functional logic at the freshdesk level

Reply:
Sounds good